pkg/kv/local/driver.go

```diff
@@ -0,0 +1,76 @@
+package local
```

Comment:
Why did you go with a different structure than the one we implemented for all other drivers?
I think it will be confusing for someone from the outside (new developer or contributor), so might be better to maintain the same structure

Reply:
Please do. I think it will make much more sense keeping a consistent structure, when possible

Reply:
This is fine by me. As I said - it has no real impact and I don't think it's a blocker.
If you do believe it makes more sense the way it is currently organized, maybe we should refactor the rest of the drivers to match that too? (not as part of this PR, of course...)